Terminal.Gui/Drawing/Color/Color.ColorExtensions.cs

```diff
@@ -45,7 +45,7 @@ static ColorExtensions ()
             { new Color(255, 0, 255), ColorName16.Magenta },
             { new Color(255, 255, 0), ColorName16.Yellow },
             { new Color(128, 128, 128), ColorName16.Gray },
-            { new Color(118, 118, 118), ColorName16.DarkGray },
+            { new Color(169, 169, 169), ColorName16.DarkGray },
```

Comment:
That correspond adding 20% of the opacity (118+255*0.20=118+51=169), which may be the opacity percentage used by the `StandardColor`? It's possible to comprove with a unit test the opacity percentage (20%) for all `StandardColor`?